terraform/azure/ha-availability-set-new-vnet/README.md

```diff
@@ -98,6 +120,14 @@ This service principal will be used by Terraform in order to deploy the solution
     bootstrap_script                = "touch /home/admin/bootstrap.txt; echo 'hello_world' > /home/admin/bootstrap.txt"
     allow_upload_download           = true
     disable_password_authentication = false
+##Revision History
+In order to get template version refer to the [sk116585](https://supportcenter.checkpoint.com/supportcenter/portal?eventSubmit_doGoviewsolutiondetails=&solutionid=sk116585)
```

Comment:
In order to check the template version refer to sk...

Reply:
Done